examples/send_from_file/example.exs

```diff
@@ -202,13 +251,16 @@ defmodule Peer do
 
   defp handle_webrtc_message({:connection_state_change, :connected} = msg, state) do
     Logger.info("#{inspect(msg)}")
-    Logger.info("Starting sending video.ivf")
+    Logger.info("Starting sending video.ivf and audio.ogg...")
     {:ok, ivf_reader} = IVFReader.open("./video.ivf")
     {:ok, _header} = IVFReader.read_header(ivf_reader)
-    payloader = VP8Payloader.new(800)
+    vp8_payloader = VP8Payloader.new(800)
+
+    {:ok, ogg_reader} = OggReader.open("./audio.ogg")
 
-    Process.send_after(self(), :send_frame, 30)
-    %{state | ivf_reader: ivf_reader, payloader: payloader}
+    Process.send_after(self(), :send_video_frame, 30)
+    Process.send_after(self(), :send_audio_packet, 20)
```

Comment:
Actually, we could do Process.sendafter 0

Reply:
Even better, `send(self(), ...)` 🙂